provisioner/template.go

```diff
@@ -647,3 +648,16 @@ func extractEndpointHosts(endpoints string) ([]string, error) {
 	sort.Strings(result)
 	return result, nil
 }
+
+func indexedList(itemTemplate string, length int) (string, error) {
```

Comment:
The `length` argument will come from the config items and all the values there are strings. You want to either accept a string argument here, and do the conversion inside, or change it to `int64` because we already have a `parseInt64` function.